packages/identity-service/src/audiusLibsInstance.js

```diff
@@ -20,18 +21,11 @@ class AudiusLibsWrapper {
   async init() {
     let web3Config
     if (config.get('environment') !== 'development') {
-      const dataWeb3 = await AudiusLibs.Utils.configureWeb3(
-        web3ProviderUrl,
-        null,
-        false
-      )
-      if (!dataWeb3) throw new Error('Web3 incorrectly configured')
-
       web3Config = {
         registryAddress,
         useExternalWeb3: true,
         externalWeb3Config: {
-          web3: dataWeb3,
+          web3: new Web3(web3ProviderUrl),
```

Comment:
what's this all about

Reply:
see thread in stage-triage. it's broken because of the above throw

Reply:
since /nethermind no longer exists on stage